schemas/openapi.yml

```diff
@@ -20,6 +23,12 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Task'
+          x-postman-variables:
```

Comment:
Thanks to the work of @allenheltondev our OpenAPI spec can automatically generate tests that are very involved.

One of these tests involves saving an ID when hitting the POST endpoint, and then reusing it later on.

I don't like that the header has "postman" in the name. Not because I don't like Postman, but because this is supposed to be a very agnostic OpenAPI spec.
So I will discuss with Allen so we can change the name. I don't want to start forking his work, because we need to work on improving this automatically generated test suite, and we have to work together to do so.